packages/runner/injection/main.js

```diff
@@ -16,6 +16,19 @@ if (!Cypress) {
 Cypress in the parent window but it is missing. This should never happen and likely is a bug. Please open an issue.')
 }
 
+/* eslint-disable */
+const originalSetAttribute = Element.prototype.setAttribute
+
+Element.prototype.setAttribute = function (qualifiedName, value) {
+  if (qualifiedName === 'integrity' && Cypress.config('useExpandedModifyObstructiveCode')) {
+    qualifiedName = 'cypress-stripped-integrity'
+  }
+
+  originalSetAttribute.apply(this, [qualifiedName, value])
```

Comment:
```suggestion
  return originalSetAttribute.apply(this, [qualifiedName, value])
```

Reply:
addressed in c360238ed65cdca535ac55d7e1f3eb24e94889e1